internal/mocks/thrift/utils.go

```diff
@@ -62,5 +62,9 @@ func ToApacheCodec(p thrift.FastCodec) athrift.TStruct {
 
 // UnpackApacheCodec unpacks ToApacheCodec
```

Comment:
The comment should be "UnpackApacheCodec unpacks  **ApacheCodec**" ?

Reply:
// UnpackApacheCodec unpacks the value returned by `ToApacheCodec`